src/components/LoanCards/Buttons/LendCtaExp.vue

```diff
@@ -196,6 +204,17 @@ export default {
 		unreservedAmount() {
 			return this.loan?.unreservedAmount ?? '';
 		},
+		amountLeft() {
+			if (this.enableRelendingExp) {
+				if (this.enableFiveDollarsNotes) {
+					if (this.userBalance > 20) return Number(this.unreservedAmount) > 25 ? '25' : this.unreservedAmount;
+					return this.unreservedAmount > 5 ? '5' : this.unreservedAmount;
+				}
+				if (this.unreservedAmount > 25) return Number(this.unreservedAmount) > 5 ? '5' : this.unreservedAmount;
```

Comment:
Do we need this line? If so, what's the use case?

Reply:
yeah. my bad, this is for case outside $5 notes test ...